doc/release-notes.md

```diff
@@ -1,348 +1,296 @@
+  which caused problems with the `sendrawtransaction` RPC call. (#3088)
+* Fixed the example `dogecoind.service` file (#3066)
+* Fixed a bug where misnamed signal handlers were causing excessive warnings
+  in Dogecoin Qt logs (#3063)
+* Fixed a bug where extremely long wallet labels could cause Qt popups to flow
+  over screen limits, causing the user to have no means of accepting or
+  rejecting the transaction (#3224)
+* Fixed an interaction error where running Qt tests interacted with the mainnet
+  datadir (#3286)
+* Fixed a bug that caused qt4 compilation to fail (#3427)
+
 
 Minor Changes
 =============
 
-* Added ARMv8 and ARMv82 crypto extensions as an experimental feature. This
-  significantly improves the speed at which 64-bit ARM hardware can validate
-  blocks and transactions.
-* Fixed the graphical wallet to properly display in High-DPI resolutions.
-* Improved privacy by preventing leaks of information about our wallet when
-  setting the locktime field (automatically.)
-* Added scrypt to the benchmark code.
-* Change the granularity of data downloaded in the network tab to no longer
-  display gigabytes and terrabytes, as this obscures too much detail. Instead,
-  the largest unit is now megabytes.
-* Fixed a bug in the getutxos REST endpoint to also show data when not
-  including the mempool.
-* Improved the tips displayed in the UI to be more concise and give better
-  advice.
-* Removed information from the graphical wallet that was attempting to indicate
-  stuck transactions based on peer-to-peer traffic, but this does not work
-  since 1.14.2, and can be misleading (showing false positives and negatives.)
-* Improved and updated the [bip list](bips.md) documentation that explains the
-  protocol features implemented by Dogecoin Core.
-* Greatly improved the `gitian-build` script to ease doing deterministic builds
-  for everyone, and the [documentation](gitian-building.md) of that process.
+* Enforced explicit enabling of experimental features by introducing a build
+  flag (`--enable-experimental`) and enforce these at compile time (#3136).
+  See [experiments.md](experiments.md) for more information about which
+  experiments are available to self-compile.
+* Re-enabled Scrypt SSE2 routines and encapsulated it as an experimental feature
+  to help speeding up PoW verification on x86_64 architecture. (#2773)
+* Packaged the Noto Sans font to make sure all languages can be rendered (#2676)
+* Updated `assumevalid` to help new nodes save CPU time by setting it to block
+  `e7d4577405223918491477db725a393bcfc349d8ee63b0a4fde23cbfbfd81dea`, at height
+  5,050,000 on mainnet (#3416)
+* Enhanced the paper wallet print function to default to the User's paper format
+  instead of A4 (#3239)
+* Updated build guides for MacOS (#2686), Windows (#3340, #3411), Fedora (#3434)
+* Introduced modern tooling for inspecting and constraining our binaries (#3204)
+* Refactored out C-style NUL-terminated strings in interfaces (#3157)
```

Comment:
NULL is missing an L

Reply:
`NUL` == `\0`, to my knowledge this is actually the correct name when it comes to C character arrays that are terminated with `\0`.

Reply:
> NULL is missing an L

NULL has Tomahawk